llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -1301,6 +1407,11 @@ Context::createGetElementPtrInst(llvm::GetElementPtrInst *I) {
   return cast<GetElementPtrInst>(registerValue(std::move(NewPtr)));
 }
 
+PHINode *Context::createPHINode(llvm::PHINode *I) {
+  auto NewPtr = std::unique_ptr<PHINode>(new PHINode(I, *this));
+  return cast<PHINode>(registerValue(std::move(NewPtr)));
```

Comment:
nit: Perehaps `return cast<PHINode>(registerValue(std::unique_ptr<PHINode>(new PHINode(I, *this))));` ?

Reply:
That ends up being different from the others, like on the code just above this one. Should we change them all?

Reply:
Sure, your call. Either is fine with me.

Reply:
I mean we can keep it as it is too, I don't feel strongly about changing them.